### PR TITLE
Adding Scrapy Badges to Home, Download and Doc pages

### PR DESCRIPTION
**Pull request by giopiazza**
Badges as seen in the [Scrapy repo](https://github.com/scrapy/scrapy).

Screenshots:

Home Page
---
captura de tela 2015-08-20 as 20 35 39

Download Page
---
captura de tela 2015-08-20 as 20 36 06

Documentation Page
---
captura de tela 2015-08-20 as 20 36 29

### CODE REVIEW

**Comment by cyberplant** [comment]
:+1:

**Comment by pablohoffman** [comment]
Looks good there, but can we put the code in a single file that we include in all places?. To avoid repeating the same code in the 3 files.

**Comment by giopiazza** [comment]
Sure, I'll update that! Thanks @pablohoffman , that way it'll be easier to maintain right? :)

**Comment by giopiazza** [comment]
@pablohoffman updated  :+1: :wink:

File: _includes/badges-bar.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  </a>
+  <a href="https://pypi.python.org/pypi/Scrapy">
+    <img alt="Wheel Status" src="https://camo.githubusercontent.com/609890facf66fde6b5065196cde85ed95efa1c43/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f776865656c2d7965732d627269676874677265656e2e737667" data-canonical-src="https://img.shields.io/badge/wheel-yes-brightgreen.svg" style="max-width:100%;">
+  </a>
+  <a href="https://github.com/scrapy/scrapy/wiki/Python-3-Porting">
+    <img alt="Python 3 Porting Status" src="https://camo.githubusercontent.com/4435e82dc7497e4e896e70483c0233ea3df6ce5d/687474703a2f2f7374617469632e7363726170792e6f72672f70793370726f67726573732f62616467652e737667" data-canonical-src="http://static.scrapy.org/py3progress/badge.svg" style="max-width:100%;">
+  </a>
+  <a href="http://codecov.io/github/scrapy/scrapy?branch=master">
+    <img alt="Coverage report" src="https://camo.githubusercontent.com/e8aa0afb265628547b91b7fd8b375cb055960860/68747470733a2f2f696d672e736869656c64732e696f2f636f6465636f762f632f6769746875622f7363726170792f7363726170792f6d61737465722e737667" data-canonical-src="https://img.shields.io/codecov/c/github/scrapy/scrapy/master.svg" style="max-width:100%;">
```
Comment (dangra) [inline _includes/badges-bar.html]: `src` attribute is using GH content distribution network, the real link must be the one in `data-canonical-src` attribute.

Comment (pablohoffman) [inline _includes/badges-bar.html]: @giopiazza what @dangra means is to update the content of `src` with the value of `data-canonical-src`

Comment (pablohoffman) [inline _includes/badges-bar.html]: @giopiazza I merged it and will update myself, nvm!

Comment (dangra) [inline _includes/badges-bar.html]: :+1:

Comment (giopiazza) [inline _includes/badges-bar.html]: Oh, thanks guys! Sorry about that

**Pull request merged**